File: src/jdbc/IP.java
package jdbc;


/**
 * Created by jacobmenke on 7/14/17.
 */
public class IP {
    public static  String IPADDR = "";
    public static String passwd = "";
    public static String port = "3306";

    public static String getIP() {
        return IPADDR;
    }

    public static void setIP(String ip) {
        IP.IPADDR = ip;
    }

    public String getPasswd() {
        return passwd;
    }

    public static void setPasswd(String passwd) {
        IP.passwd = passwd;
    }

    public String getPort() {
        return port;
    }

    public void setPort(String port) {
        IP.port = port;
    }
}

File: src/jdbc/IP.kt
package jdbc

class IP {
    companion object {
        @JvmStatic
        var ipAddr: String = ""

        @JvmStatic
        @get:JvmName("getPasswd_")
        @set:JvmName("setPasswd_")
        var passwd: String = ""

        @JvmStatic
        @get:JvmName("getPort_")
        @set:JvmName("setPort_")
        var port: String = "3306"
    }

    fun getIp(): String = ipAddr
    fun setIp(ip: String) { ipAddr = ip }
    fun getPasswd(): String = passwd
    fun setPasswd(passwd: String) { IP.passwd = passwd }
    fun getPort(): String = port
    fun setPort(port: String) { IP.port = port }
}
